Gemfile

```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org'
 
 gemspec
+gem 'rake', '< 11.0'
```

Comment:
1. Any reason you chose to do this in the `Gemfile` instead of in the `gemspec`? There's currently a `rake` dependency defined in the `gemspec` already: https://github.com/ruby-hl7/ruby-hl7/blob/master/ruby-hl7.gemspec#L33
2. What's using `last_comment` and is it possible to fix that instead?